bugsnag-process.md

```diff
@@ -0,0 +1,149 @@
+# Bugsnag Process
+
+## tl;dr
+- check Bugsnag regularly
+- check metrics in the morning and during the day
+- send an email at the end of the day with info about the above
+
+---------------
+
+The Bugsnag duties rotate from developer to developer on a weekly basis.
+
+Bugsnag is an important part of our developer role here. It allows us time to catch and resolve bugs and issues which we would not otherwise see or prioritise. This creates a better development environment for us all as well as improving the customers experience on their site. 
+
+No one likes a buggy website, so why should we build one?
+
+During your Bugsnag week you are removed from your usual sprint activities and instead perform the following tasks to maintain the health of our sites. If there are no other Bugsnag duties for you to perform you may continue with sprint work for that day.
+
+The key activities you should perform are:
+
+- checking Bugsnag for bugs being thrown
+- checking HXMetrics for spikes in error rates
+- sending an email out at the end of every day to update people on the above
+
+There is also a [Bugsnag Kanban board](https://holidayextras.jira.com/secure/RapidBoard.jspa?rapidView=284) which has tasks related to known bugs which you should attempt to work through in order to reduce our overall bug count.
+
+The most important part of Bugsnag is communication. You should be talking to people about what you're seeing and updating on things you're working on etc. Don't suffer in silence on a tough bug, ask for help. And don't assume an issue is being looked at, check!
+
+## Bugsnag
+
+You'll require our Bugsnag credentials. Ask the last person on bugsnag for these.
+
+### How do I find bugs?
+Errors on our site are reported to [Bugsnag](https://bugsnag.com/holiday-extras/trip-app-js/errors) and are visible in the dashboard.
+
+The errors are split by project and you can switch between these using the menu in the top left corner.
+
+![](https://cldup.com/BjzztadbNV-1200x1200.png)
```

Comment:
I recommend attaching the screenshots to this PR and linking from there, this way all the content will be hosted in GitHub without external dependencies.